release/bin/start.sh

```diff
@@ -16,9 +16,6 @@
 
 set -e
 
-ulimit -c unlimited
-ulimit -n 655360
```

Comment:
noted on the `limits.conf`.

However, `ulimit -n 655360` always says `Operation not permitted`. I think it's because the value exceeds the current settings.
![image](https://user-images.githubusercontent.com/7508922/206335246-3a78ab06-3466-47a9-98d1-fe6a5d5cf687.png)